.github/workflows/reuse.sqllogic.yml

```diff
@@ -279,6 +279,9 @@ jobs:
         dedup:
           - "full_path"
           - "sub_path"
+        size:
+          - "small"
+          - "large"
```

Comment:
Cc @everpcpc please review this. I'm not sure the size is ok.

Reply:
these are local self-defined names, what maybe the problem?

works as expected 

![image](https://github.com/user-attachments/assets/786049b3-2530-4b68-8ffb-1ce08b91c6f3)